models/articles.js

```diff
@@ -21,7 +21,19 @@ exports.fetchArticles = () => {
 
 exports.fetchArticleById = (article_id) => {
   return db
-    .query("SELECT * FROM articles WHERE article_id = $1;", [article_id])
+    .query(
+      `
+    SELECT articles.author, title, articles.article_id,
+        topic, articles.created_at, articles.votes, articles.body,
+        COUNT(comments.body) AS comment_count
+    FROM articles
+    LEFT JOIN comments ON articles.article_id = comments.article_id
+    WHERE articles.article_id = $1
+    GROUP BY articles.author, title, articles.article_id,
+        topic, articles.created_at, articles.votes;
```

Comment:
this group by does not need to have this many column names this can be refactored down to using a single one to group everything together.